File: TestMessenger/Entity/TokenRecord.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AuthenticationService.Entity
{
    internal class TokenRecord
    {
        public long id { get; set; }
        public long user_id { get; set; }
        public string token { get; set; }
        public string ExpiresAt { get; set; }
        public string CreatedAt { get; set; }
    }
}

File: AuthenticationService/src/main/java/ru/TestMessenger/AuthenticationService/Model/TokenRecord.java
package ru.TestMessenger.AuthenticationService.Model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.Date;

@Data // добавляет get set
@Entity
@Table(name = "token_records", schema = "auth_schema")
public class TokenRecord {

    @Id
    @JsonIgnore
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @OneToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id", nullable = false)
    @JsonIgnore
    private User user;

    @Column(nullable = false, unique = true)
    private String tokenValue;

    @Column(name = "expires_at")
    private LocalDateTime expirationDate;

    @Column(name = "created_at")
    private LocalDateTime createdDate;
}